draft-voit-rats-attestation-results.md

```diff
@@ -200,18 +200,28 @@ Additionally, the Relying Party must have confidence that the Trustworthiness Cl
 
 ### Attesting Environment
 
-For the Attesting Environment identity, there MUST exist a chain of trust ultimately bound to a hardware-based root of trust in the Attesting Environment.
+This document recognizes three general categories of Attesting Environments.
+
+1. TPM-based: A cryptoprocessor exists which continually hashes security measurements in a way which prevents an Attester from lying about measurements taken since boot (e.g., TPM2.0.)
+2. Process-based: Individual processes which has its runtime memory encrypted in a way that no other processes can read and decrypt that memory (e.g., {{SGX}} or {{-PSA}}.)
+3. VM-based: An entire Guest VM (or a set of containers within a host) have been encrypted as a unit so that the host operating system cannot read and decrypt what is executing within that VM (e.g., SEV or TDX.)
+
+Each of these three types of Attesting Environments must generate Evidence which is protected using private keys or certificates which are not accessible outside of that Attesting Environment.  
```

Comment:
I think we also need to cover the case where the Evidence produced by the AE is not directly signed by that AE, but by another AE that has a "trusted path" to the first one.

Reply:
I agree that this method of packaging Evidence generated elsewhere in the system is a viable one which needs to be covered. (in fact that is what the TPM does.)  So unless I am misunderstanding, I don't think this is a new case of technology separated from the other three.  Instead it is important that an vulnerabilities which are exposed by the 'proxy' or 'MITM' function are understood and covered in the other two cases.  Agree?  If yes, we can craft some text.

Reply:
I checked with the Fraunhofer competency cluster and got this back as a counter example that aligns with Thomas' point.

https://cpl.thalesgroup.com/encryption/hardware-security-modules

The Internet seems to disagree with itself here. Which reduces the viability of "HSM" here, a bit. Which... also does not make this easier.

If it is neither e(SE) nor HSM, what is it? Any other thoughts?

Reply:
I agree.  Based on above, I have rewritten the text extensively.  Take a look at the new pull request.